### PR TITLE
some bureaucracy sprite fixes

### PR DESCRIPTION
**Pull request by alsoandanswer**
and fixes writing on cig filters

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Comment by Geevies** [comment]
Wait, with attackby being returned early, can filtered cigs still get lit with welders and lighters?

File: code/game/objects/items/weapons/cigs_lighters.dm [hunk 1]
```diff
@@ -731,6 +731,14 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	icon_state = "cigpaper_generic"
 	w_class = 1.0
 
+/obj/item/paper/cig/filter/attackby(obj/item/P as obj, mob/user as mob)
```
Comment (Alberyk) [inline code/game/objects/items/weapons/cigs_lighters.dm]: Two attackby procs

File: code/game/objects/items/weapons/cigs_lighters.dm [hunk 2]
```diff
@@ -742,6 +750,20 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	icon_state = "cigfilter"
 	w_class = 1.0
 
+/obj/item/paper/cig/filter/attackby(obj/item/P as obj, mob/user as mob)
```
Comment (Alberyk) [inline code/game/objects/items/weapons/cigs_lighters.dm]: Two attackby procs

File: code/modules/paperwork/paper.dm [hunk 3]
```diff
@@ -535,8 +535,6 @@
 
 	else if(istype(P, /obj/item/flame) || P.iswelder())
 		burnpaper(P, user)
-	else if(P.iswelder())
-		burnpaper(P, user)
```
Comment (Alberyk) [inline code/modules/paperwork/paper.dm]: Why is this being removed?

Comment (alsoandanswer) [reply code/modules/paperwork/paper.dm]: redundant with line above

**Pull request merged**